### PR TITLE
removes space in gsub

### LINKED ISSUE #388: Making a project with capital letters causes weird spacing in console
# Issue summary
Pretty low priority issue, all the code generation seems to still be alright. Having many capital letters in a row causes the name of the project to be weirdly spaced. 

## Expected behavior
Either project names in caps should be disallowed or all-caps names should stay all-caps. 

## Actual behavior
 The name `INCAPS` spontaneously becomes `I nc ap s`
image
The issue also exists if I named it something like `typing-normally-NOWSCREAMING`
Happens somewhat if I have names in `PascalCase` (seems like this could be intentional?)
image

## Steps to reproduce the problem

1. `shopify create project TEST`

### PR DESCRIPTION
**Pull request by katiedavis**
### WHY are these changes introduced?

Fixes #388  

The space in this regex was causing capitalized titles to print with spaces in them.

### CODE REVIEW

**Comment by tylerball** [comment]
does that mean that we need another test somewhere?

**Comment by katiedavis** [comment]
> does that mean that we need another test somewhere?

Yep done!

File: test/shopify-cli/forms/create_app_test.rb [hunk 1]
```diff
@@ -220,6 +229,17 @@ def ask(name: 'test-app', title: nil, type: 'node', org_id: 42, shop: 'shop.mysh
           shop_domain: shop,
         )
       end
+
+      def ask_uppercase(name: 'TEST-APP', title: nil, type: 'node', org_id: 42, shop: 'shop.myshopify.com')
```
Comment (tanema) [inline test/shopify-cli/forms/create_app_test.rb]: You could have just used ask and did `ask(name: 'TEST-APP')`

File: lib/shopify-cli/forms/create_app.rb [hunk 2]
```diff
@@ -17,7 +17,7 @@ def ask
       private
 
       def fallback_title
-        name.gsub(/(.)([A-Z])/, '\1 \2') # change camelcase to title
+        name.gsub(/(.)([A-Z])/, '\1\2') # change camelcase to title
```
Comment (tanema) [inline lib/shopify-cli/forms/create_app.rb]: This is mean to check for a capital letter with something proceeding it then add a space in-between. You have taken out the space so essentially this does nothing now. The tests still passed after this?

**Pull request merged**